lxc/launch.go

```diff
@@ -96,7 +96,7 @@ func (c *launchCmd) run(config *lxd.Config, args []string) error {
 		}
 
 		if count != 2 {
-			return fmt.Errorf(gettext.Gettext("bad number of things scanned from resource"))
+			return fmt.Errorf(gettext.Gettext("bad number of things scanned from image, ccontainer or snapshot"))
```

Comment:
And again (copy paste, I suspect :) )

Also, is "things" really what we want displayed?